src/main/java/edu/cornell/kfs/sys/web/CuWebUtilities.java

```diff
@@ -107,4 +109,8 @@ public static long convertLocalDateTimeToMilliseconds(LocalDateTime localDateTim
         return milliseconds;
     }
 
+    public static String urlEncode(final String value) {
+        return URLEncoder.encode(value, StandardCharsets.UTF_8);
```

Comment:
does this need some error handling / null checks?
I think URLEncoder.encode will throw an error if the String is null.